src/components/AuthDisplay/AuthDisplay.stories.jsx

```diff
@@ -0,0 +1,21 @@
+// react libraries
+import * as React from 'react';
+import { Provider } from 'react-redux';
+
+// third-party libraries
+import { storiesOf } from '@storybook/react';
+
+import store from 'store/';
+
+// components
+import { AuthDisplay } from './index';
+
+const connectedComponent = (form) => (
```

Comment:
We are not writing stories for connected components. Please review this

Reply:
> We are not writing stories for connected components. Please review this

Okay, I will remove all connected components from storybook.
But then, I actually like using it as so since it makes development easy and eliminates the need to import components in the app in other to see them on the browser. I am removing them all.

Reply:
I don't understand? Were you on the call when we made a decision to only write stories for reusable components?

Reply:
What does AuthDisplay do?

Reply:
I

> What does AuthDisplay do?

It is the container for auth forms. e.g login, signup and social login.
The modal has been improved to accept props and display based on what it recieves. the AuthDisplay is the auth form. Other components can be created and passed to the modal and the modal will render them appropraitely.